src/containers/trends/TrendsSVGContainer.js

```diff
@@ -105,20 +109,13 @@ export class TrendsSVGContainer extends React.Component {
 
   renderCbg() {
     if (this.props.showingCbg) {
```

Comment:
Nope! One of the <em>really</em> important things about using TransitionMotion to do the mount, enter, and exit animations, is that the TransitionMotion container has to still be rendered in order for those "transition" animations to happen. So the smbg side still has the check because I haven't updated the TransitionMotion usage there (yet). Now the check has moved lower - <em>inside</em> TransitionMotion [here](https://github.com/tidepool-org/viz/blob/jebeck/trends-cbgslice-changes/src/components/trends/cbg/CBGSliceAnimated.js#L216-L218).